content/en/docs/collector/building/receiver.md

```diff
@@ -397,8 +397,8 @@ value for it so it can be used as part of the default settings.
 Go ahead and add the following code to your `factory.go` file:
 
 ```go
-const (
-	typeStr         = "tailtracer"
+var (
+	typeStr         = component.MustNewType("tailtracer")
 	defaultInterval = 1 * time.Minute
 )
```

Comment:
nit: We could leave `defaultInterval` as `const`

Reply:
Fixed! Can you take a second look?